node/src/accountant/payment_adjuster/preparatory_analyser/mod.rs

```diff
@@ -398,23 +394,20 @@ mod tests {
         test_name: &str,
         disqualification_gauge: DisqualificationGaugeMock,
         original_accounts: [QualifiedPayableAccount; 2],
-        cw_service_fee_balance: u128,
+        cw_service_fee_balance_minor: u128,
     ) {
         init_test_logging();
         let determine_limit_params_arc = Arc::new(Mutex::new(vec![]));
-        let disqualification_gauge = double_mock_results_queue(disqualification_gauge)
-            .determine_limit_params(&determine_limit_params_arc);
+        let disqualification_gauge =
+            make_mock_with_two_results_doubled_into_four(disqualification_gauge)
```

Comment:
Maybe you don't need to double them into 4, you might win this test even with two results.